components/inputs/docs/input-percent.md

```diff
@@ -0,0 +1,50 @@
+# Percent Inputs
+
+The `<d2l-input-percent>` element is similar to `<d2l-input-number>`, except it provides a "%" symbol beside the number.
+
+![example screenshot of number input](../screenshots/percent.png?raw=true)
+
+```html
+<script type="module">
+  import '@brightspace-ui/core/components/inputs/input-percent.js';
+</script>
+<d2l-input-percent label="Label" value="100"></d2l-input-percent>
+```
+
+**Properties:**
+
+| Property | Type | Description |
+|--|--|--|
+| `label` | String, required | Label for the input. |
+| `autocomplete` | String | Specifies which types of values [can be autofilled](https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/autocomplete) by the browser. |
```

Comment:
`autocomplete` can likely be excluded since none of the value types that are numbers (like birthday-month) would apply to percents.